lib/Twig/Node/SetTemp.php

```diff
@@ -33,3 +33,5 @@ public function compile(Twig_Compiler $compiler)
         ;
     }
 }
+
+class_alias('Twig_Node_SetTemp', 'Twig\Node\SetTempNode', false);
```

Comment:
just wondering: this class has been removed in 2.x
why? missing deprecation notice in 1.x or mistake in 2.x?